db/seeds/dev/03_weapons.js

```diff
@@ -0,0 +1,33 @@
+const weaponsData = require('../../../data/weapons');
+
+const getClassID = async (knex, weapon) => {
+  const className = await knex('classes').where({
+    name: weapon.class
+  }).returning('id');
+  return className[0]['id'];
+}
```

Comment:
Name of class for any from dataset  is changed by id that's defined for this class in `classes` table of DB